karspexet/templates/menu.html

```diff
@@ -0,0 +1,15 @@
+{% load staticfiles fontawesome%}
+{% for child in children %}
+    <li class="{%if forloop.last%}menu_ticket_link {% endif %} child{% if child.selected %} selected{% endif %}{% if child.ancestor %} ancestor{% endif %}{% if child.sibling %} sibling{% endif %}{% if child.descendant %} descendant{% endif %}">
```

Comment:
OK, so this makes me a bit sad. @Frost can this be extracted somehow to avoid all that logic inside the template? (I haven't used Django's templates myself)

Reply:
This code snippet is mostly taken directly from https://github.com/divio/django-cms/blob/develop/menus/templates/menu/menu.html...

Reply:
😿

Reply:
Yeah, we can probably do that using a custom template tag instead. I'll have a look at it tomorrow or so.

Reply:
The `{% if forloop.last %}` part can be done entirely in CSS instead, so we can have something like:

```css
navbar ul li:last-child {
  // whatever rules the .menu_ticket_link css rule has
}
```

The rest of those rules I guess come from django-cms. If I were to implement that kind of thing, I would probably do it using data-attributes on the tag instead of if-statements in the class-attribute, but since that comes from a framework, we'll have to handle it some other way.